src/app/pages/signing-auth-redirect/signing-auth-redirect.component.ts

```diff
@@ -13,7 +13,7 @@ export class SigningAuthRedirectComponent implements OnInit {
 
   goToSigning() {
     // noinspection JSIgnoredPromiseFromCall
-    this.authService.login('/signing', { uid: this.uid, key: this.key });
+    this.authService.login('signing', { uid: this.uid, key: this.key });
```

Comment:
Incorrect usage of routerLink
```suggestion
    this.authService.login('/signing', { uid: this.uid, key: this.key });
```